.github/workflows/ci.yml

```diff
@@ -73,8 +75,10 @@ jobs:
       - name: README check
         if: runner.os == 'Linux'
         run: |
-          if [[ -n $(git diff --name-only main..HEAD README.md) ]]; then
-            echo "README.md was updated. Please update README.md.in instead."
+          # Requires the checkout to be not shallow.
+          BASE_COMMIT=$(git merge-base HEAD origin/main)  # Find the original split point.
+          if ! git diff --quiet $BASE_COMMIT -- README.md; then
+            echo "::error::README.md was modified in this branch compared to its original base commit."
```

Comment:
'modified ... compared to' sounds a bit weird.

Maybe 'README.md on this branch differs from its original base commit.'?

Reply:
done.